supply-chain/audits.toml

```diff
@@ -69,37 +69,37 @@ notes = "I am the author of this crate."
 [[audits.cap-fs-ext]]
 who = "Dan Gohman <dev@sunfishcode.online>"
 criteria = "safe-to-deploy"
-version = "0.26.0"
+version = "1.0.1"
 notes = "The Bytecode Alliance is the author of this crate"
 
 [[audits.cap-primitives]]
 who = "Dan Gohman <dev@sunfishcode.online>"
 criteria = "safe-to-deploy"
-version = "0.26.0"
+version = "1.0.1"
 notes = "The Bytecode Alliance is the author of this crate"
 
 [[audits.cap-rand]]
 who = "Alex Crichton <alex@alexcrichton.com>"
 criteria = "safe-to-deploy"
-version = "0.26.0"
+version = "1.0.1"
 notes = "The Bytecode Alliance is the author of this crate"
 
 [[audits.cap-std]]
 who = "Dan Gohman <dev@sunfishcode.online>"
 criteria = "safe-to-deploy"
-version = "0.26.0"
+version = "1.0.1"
 notes = "The Bytecode Alliance is the author of this crate"
 
 [[audits.cap-tempfile]]
 who = "Dan Gohman <dev@sunfishcode.online>"
 criteria = "safe-to-run"
-version = "0.26.0"
+version = "1.0.1"
```

Comment:
I believe `audits.toml` is intended to be an append-only log, so could this add new audits instead of replacing existing ones?

Reply:
Done